WIP: Add asd file

jgarte opened the pull request:
Closes #17, #28 and #29

jgarte committed:
Add asd file

jgarte committed:
Remove hardcoded quicklisp API code.

hellerve commented:
Cool! Could we still keep the install script around and potentially just move the deleted code into its own file that loads the file here? That way both install methods work.

jgarte commented:
> 

@hellerve Sure, I updated the PR to remove that commit. I'll need to do some more testing on the asd file I added here. It's still currently a work in progress but wanted to share my work so far here in case lispers in the community would like to give feedback and code review.

hellerve commented:
I tried to make ASD work and introduce a package file, but all of the package managers simply have too many different opinions about the world and the reality is that then more than half of the REPL file becomes unscrutable package manager wrangling code.

I’ll leave this open. Maybe someone can propose a real solution, but I can only come up with garbage.
